### DISCUSSION TITLE
Write access for: @MitchellBerend

### DISCUSSION THREAD

**Discussion by chshersh** [chshersh/tool-sync#66] [Commit bits] [answered]
Hey, @MitchellBerend, according the [CONTRIBUTING](https://github.com/chshersh/tool-sync/blob/main/CONTRIBUTING.md#write-access-to-the-repository) guide for `tool-sync`, you can request Write Access to this repository. Maybe the wording is not that clear or maybe you don't want extra responsibility 😌

But I see that you've already contributed a lot to `tool-sync` so let me know if you would like to have write access (in case it can simplify your workflow).

**Comment by MitchellBerend** [comment, marked as answer]
Hi @chshersh im not quite sure if I want this responsibility. Im not sure what value this would add for myself besides adding labels to issues/prs myself. I currently work on branches in my own fork and I would like all my proposed changes to be reviewed anyway to make sure its actually up to snuff.

I wanted to thank you for the reviews and suggestions btw. I appreciate the way you do those.

**Comment by chshersh** [comment]
Thanks for your input, @MitchellBerend! At this point there's no actual responsibility. I guess, write access makes it more convenient to work with the repo so you don't need to fork.

But anyway, feel free to request access at any time when you would like to have it 😌

**Comment by MitchellBerend** [comment]
I will abuse this thread to thank you for the work you are doing and constructive feedback you are giving btw. I appreciate the way its done and I also really like the way the issues are small in scope and numerous. 

Thank you for that.

**Comment by MitchellBerend** [comment]
Actually, is there a way to just give me label rights? Those would actually be helpful.

**Comment by chshersh** [comment]
@MitchellBerend I've sent you an invite. IIRC, there's the "Triage" permission level to enable the ability to add labels. But I can't set it until the invite is accepted 🤔

**Comment by MitchellBerend** [comment]
Okay I accepted the invite, thanks for the response!
